src/main/webapp/formNewCategory.jsp

```diff
@@ -0,0 +1,79 @@
+<%@ page pageEncoding="UTF-8" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<c:url value="/criaCategoria" var="linkServletNovaCategoria"/>
+<html>
+<head>
+    <title>Cadastro da Categoria</title>
+</head>
+<body>
+
+    <h1>Cadastro da Categoria: </h1>
+    <form action="${linkServletNovaCategoria}" method="post">
+
+        <fieldset>
+            <legend>Dados de categorias</legend>
+            <table>
+                <tr>
+                    <td>
+                        <label>Nome: </label>
```

Comment:
o correto é que os arquivos `.jsp` fiquem dentro da pasta`WEB-INF` para o cliente não acessar diretamente. geralmente deixamos em `WEB-INF/views` (***request changed**)

Reply:
quando coloco esta classe no caminho WEB-INF/views ela não é encontrada msm alterando o caminho no href, para as outras deu certo (mas o redirecionamento delas é a partir de uma servlet)
pego esse erro -> The origin server did not find a current representation for the target resource or is not willing to disclose that one exists <- ao clicar no href cadastrar

Reply:
exato, tudo que fica na pasta `/WEB-INF` o cliente não pode acessar. como um JSP não é um arquivo estático e dinâmico e é compilado pelo Jasper dentro do Tomcat, deve ficar nesta pasta em que apenas o servidor acessa. o correto é você fazer uma Servlet mepada com uma rota apenas para expor esta JSP

Reply:
entendi, obrigado.